examples/exercises/plot_cv_diabetes.py

```diff
@@ -33,10 +34,16 @@
 clf.fit(X, y)
 scores = clf.cv_results_["mean_test_score"]
 scores_std = clf.cv_results_["std_test_score"]
+
 plt.figure().set_size_inches(8, 6)
 plt.semilogx(alphas, scores)
+plt.xlabel("alphas")
+plt.ylabel("scores")
+plt.show()
 
-# plot error lines showing +/- std. errors of the scores
+# %%
+# Plot error lines showing +/- std. errors of the scores
+# ------------------------------------------------------
```

Comment:
I would move this header above `plt.figure().set_size_inches(8, 6)` i.e., the first appearance of the plot settings.

Reply:
I think the first plot shows the scores which doesn't seem to fit under the header. the header reads ```showing +/- error rates of the scores``` Should i still make the changes?

Reply:
Sorry for the misunderstanding. Maybe the following suggested changes make more sense, as they will make a single plot of the error line with the corresponding error bands

Reply:
Ok now i understand, i've made the changes.